docs/data-sources/services.md

```diff
@@ -46,12 +45,13 @@ data "firehydrant_services" "managed-true-labeled-services" {
 <a id="nestedatt--services"></a>
 ### Nested Schema for `services`
 
+<<<<<<< HEAD
```

Comment:
Merge conflict leftover?

Reply:
yep 🤦🏼‍♀️ ty for catching that! should be fixed now